tests/test_mutation_testing_template.py

```
@@ -0,0 +1,432 @@
+        # Render the enabled pure variant under a non-default package name so
+        # the package-specific guidance, mutmut source paths, and workflow
+        # inputs pin the template's substitution rather than a fixed directory.
+        ("mutation-313-pure", False, "3.13", True, "custom_mutation_pkg"),
+        ("mutation-314-rust", True, "3.14", True, "mutation_pkg"),
+    ],
+)
+def test_generated_mutation_testing_gating(
+    copier: CopierFixture,
+    tmp_path: Path,
+    target_dir: str,
+    use_rust: bool,
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)